### ISSUE TITLE
Results visualization

### ISSUE THREAD

**Issue by yhx-coder** [BNN-UPC/ignnition#126]
I am a new user of the ignnition framework and I would like to ask the following two questions:
1. Does ignnition provide a function or other way to compare the predicted loss function of different models on the same validation set?
2. Also is it possible to customize the output of the loss change image during training. Tensorboard doesn't seem to be able to alias the horizontal and vertical coordinates?

Looking forward to hearing from the community, thanks a lot!
